db/migrate/20220811093403_create_comments.rb

```diff
@@ -0,0 +1,10 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.references :post, foreign_key: true, null: false
+      t.references :user, foreign_key: true, null: false
```

Comment:
- [ ] Same here, user should be called as author based on EDR diagram:
`t.references :author, foreign_key: { to_table: 'users' }`
this will create author_id and a corresponding index

Reply:
Implemented 👍